index.html

```diff
@@ -53,5 +53,167 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
         </ul>
       </section>
     </section>
+
+    <main>
+      <section class="container-2">
+        <div class="w-container">
+          <h2 class="my-work">My Recent Works</h2>
+          <hr class="hr">
+        </div>
+        <div class="image-paragraph">
+          <img class="img-pd" src="./images/bg.png" alt="place holder">
+          <div class="multiple-column">
+           <h2 class="multi-column-h1">Multi-Post Stories</h2>
+           <p class="multi-column-det">A daily selection of privately personalized reads; no accounts or sign-ups required.
+            has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+           </p>
+           <ul class="what-i-know">
+            <li class="exp">css</li>
+            <li class="exp">html</li>
+            <li class="exp">bootstrap</li>
+            <li class="exp">Ruby</li>
+           </ul>
+           <button class="see-pro" type="button">See Project</button>
+          </div>
+        </div>
+        <div class="pro-art-major">
+          <div class="art">
+           <h3 class="art-data">Profesional Art Printing Data</h3> 
+            <p class="daily-s-con">A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <ul class="pro-t">
+            <li class="pro-">html</li>
+            <li class="pro-">bootstrap</li>
+            <li class="pro-">Ruby</li>
+          </ul>
+             <button class="see-project" type="button">See Project</button>
+        </div>
+        <div class="pro-art-major">
+          <div class="art">
+           <h4 class="art-data">Profesional Art Printing Data</h4> 
+            <p class="daily-s-con">A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the industry's standard
```

Comment:
[OPTIONAl] Please use present tense to with your commit and make them describe an action example Add index.html , Fixing linter issues and so on 
![image](https://user-images.githubusercontent.com/74744787/159114780-6ac9213b-dc59-4163-b280-02964056fb52.png)

Reply:
Okay, noted, sir!